subsys/sdfw_services/services/device_info/device_info_service.c

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/** .. include_startingpoint_device_info_source_rst */
```

Comment:
Is this needed?

Reply:
removed. thank you for pointing this out !

Reply:
I see the others have been removed, but this one is still there AFAIK.